eras/alonzo/formal-spec/utxo.tex

```diff
@@ -480,6 +480,9 @@ \subsection{The UTXOS transition system}
 
 There are two rules, corresponding to the two possible state changes of the
 UTxO state in the UTXOS transition system (Figure~\ref{fig:rules:utxo-state-upd}).
+Note here that the constants $\mathsf{ei}$ and $\mathsf{sysSt}$ are available
+inside the definition of all ledger rules. They are supplied by the consensus
```

Comment:
No, this is not true. The ledger rules are simply relations, so to make this true we'd have to break our entire theory. Those need to be provided by the contexts, even if the code does it differently.

Reply:
i can re-word this and point out it's an implementation detail (ie it's actually hard-coded)

Reply:
Hmm, if they're constants then maybe we can just stick them into the global constants?

Reply:
Also, I'm confused. Why is `slotToPOSIXTime` a constant?

Reply:
The use of EpochInfo is similar to the use of NetworkID

Reply:
And the NetworkID is a global constant, see Fig 8 of the Shelley spec